docs/source/weightvec.rst

```diff
@@ -63,6 +50,19 @@ Probability weights represent the inverse of the sampling probability for each o
     w = pweights([0.2, 0.1, 0.3])
 
 
+``Weights``
+~~~~~~~~~~~~
+
+The ``Weights`` type describes a generic weights vector which does not support all operations possible for ``FrequencyWeights``, ``AnalyticWeights`` and ``ProbabilityWeights``.
+
+.. code-block:: julia
+
+    w = Weights([1., 2., 3.])
+    w = Weights([1., 2., 3.,], 6.)
```

Comment:
If you keep the two-argument version for this constructor, you should show it for all types. Or just remove it here.